### ISSUE TITLE
Add support for diferent time zone calendars

### ISSUE THREAD

**Issue by melinerunen** [sfsam/Itsycal#35]
Hi!
Can you add the ability to show the current time into different time zones? For example here in Argentina we've UTC-3 but Germany uses UTC-1.

Thanks in advance

**Comment by sfsam** [comment]
Can you please clarify? Are you asking for Itsycal to show two clocks in different time zones?

**Comment by melinerunen** [comment]
@sfsam yes! two or more. Similar to the windows clock but more minimalistic 😃

**Comment by jcrada** [comment]
I find this feature request very useful too. Please, take a look at the Ubuntu time zones below, a feature which I was quite fond of and I miss a lot in Mac: 

![image](https://user-images.githubusercontent.com/303131/31316835-dd831710-ac91-11e7-9fea-d2308b70d42f.png)

**Comment by limingjie** [comment]
Like the timezone feature of [Clocker](https://apps.apple.com/us/app/clocker/id1056643111?mt=12), it is very useful in business while working internationally.
![image](https://user-images.githubusercontent.com/6889308/66698304-5ce03280-ed0f-11e9-910e-186bb969b4c4.png)

**Comment by espen** [comment]
This is a bit complex to make really useful, at least from my user cases. I use [The Clock](https://www.seense.com/the_clock/) in addition to Itsycal. Itsycal shows local time and calendar info. I then use The Clock to show another time zone in the menu bar and also I use it for a time zone meeting planner. Often I will have someone in time zone X ask for meeting on a specific date, then I use The Clock to scroll through times in that time zone + local time zone to find suitable time. The Clock has many features to managing time zones, meeting planner etc. Not sure how this can be made simple and still be useful in Itsycal.

**Comment by erango** [comment]
FWIW, I use Clocker for that (find it on the Mac App Store)

**Comment by vmrm** [comment]
so, no way and plans to implement it to Itsycal?

**Comment by jonathan-russo** [comment]
I'd love if support for this feature was added to Itsycal!

**Comment by vardecab** [comment]
It'd be great if we could have time-zone support here: 

![Image](https://github.com/user-attachments/assets/15248d41-d3d8-4d98-80a9-6b2333930523)

I know the parameter `z`, `Z`, `v` and `V` exist as per: https://unicode.org/reports/tr35/tr35-25.html#Date_Format_Patterns:
![Image](https://github.com/user-attachments/assets/abe59410-2a63-4316-9fb6-5f6ac0750496)

… but when it's added it just shows the timezone - there is no way to manipulate the time being shown:

![Image](https://github.com/user-attachments/assets/79b56249-635f-4c75-b91c-e690ed111aa0)


The way I see it, is to be able to write something like `MMM d, E • HH:mm z-5` and it shows the current time -5 hours or maybe as GMT-5.

And so then user can have something like this: `MMM d, E • HH:mm // HH:mm z-5` display this: `Jan 27, Mon • 23:15 // 18:15`
